Tests/CarthageKitTests/XcodeSpec.swift

```diff
@@ -380,11 +380,94 @@ class XcodeSpec: QuickSpec {
 			let result = ProjectLocator.locate(in: directoryURL.appendingPathComponent("ReactiveCocoaLayout")).first()
 			expect(result).to(beNil())
 		}
+
+		it("should build static library and place result to subdirectory") {
```

Comment:
`make swiftpm_test` fails in this test with error:
`Fatal error: Unexpectedly found nil while unwrapping an Optional value`

Reply:
Fixed by changing copy-fixtures script